tests/helpers.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+final class WPP_Tests_Helpers {
+
+	/**
+	 * This function is documented at <https://github.com/Yoast/wp-test-utils/blob/46566884fe74d2f758273efd71f6c07c7d2e4f53/src/WPIntegration/bootstrap-functions.php>.
+	 *
+	 * @return string|false The path to the WP test directory, or false if it could not be determined.
+	 */
+	public static function get_path_to_wp_test_dir() {
+		// phpcs:disable WordPress.PHP.YodaConditions.NotYoda
+		// phpcs:disable WordPress.PHP.NoSilencedErrors.Discouraged
+		/**
+		 * Normalizes all slashes in a file path to forward slashes.
+		 *
+		 * @param string $path File path.
+		 *
+		 * @return string The file path with normalized slashes.
+		 */
+		$normalize_path = static function ( $path ) {
+			return \str_replace( '\\', '/', $path );
```

Comment:
No need for backslashes for the global functions in this file, since there is no namespace.